### PR TITLE
Rl add quizz

### PR DESCRIPTION
**Pull request by Kurkos**
REFERENCE: AddQuizz component + part of AddQuizzContainer;
DESCRIPTION: Finished working on AddQuizz componnents(without REDUX connection);

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ import "bootstrap/dist/css/bootstrap.min.css";
 import App from "./App";
 import Login from "./components/Login/Login";
 import * as serviceWorker from "./serviceWorker";
+import AddQuizz from "./components/AddQuizz/AddQuizz";
```
Comment (janisknets) [inline src/index.js]: You have to import the container here and use the component in the Container

**Review by janisknets — CHANGES_REQUESTED** [review]

File: src/containers/QuizzContainer.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+class QuizzContainer extends Component {
+    constructor() {
+        super();
+        this.state = {
+            title: '',
+            description: '',
+        };
+    }
+    titleInputChange = (e) => this.setState({ title: e.target.value });
```
Comment (rokija) [inline src/containers/QuizzContainer.js]: onTitleInputChange

File: src/containers/QuizzContainer.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    descriptionInputChange = (e) => this.setState({ description: e.target.value });
```
Comment (rokija) [inline src/containers/QuizzContainer.js]: onDescriptionInputChange

File: src/App.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
 import React, { Component } from "react";
 import { Link } from "react-router-dom";
 import "./App.css";
-import MainLayout from "./components/MainLayout/MainLayout";
+import { Link } from "react-router-dom"
```
Comment (rokija) [inline src/App.js]: its imported twice

**Review by janisknets — APPROVED** [review]

**Review by rokija — APPROVED** [review]

**Pull request merged**